ec2utils/ec2uploadimg/python-ec2uploadimg.spec

```diff
@@ -18,7 +18,7 @@
 
 %define upstream_name ec2uploadimg
 Name:           python-ec2uploadimg
-Version:        2.0.0
+Version:        2.1.0
```

Comment:
Ah right, you could use positional arguments only when instancing EC2ImageUploader, if you're a bit masochistic. ;) Didn't think of that, sorry. Bumping the major version for such a minor feature seems odd. Should I move running_id to the end of the argument list? It would break alphabetical order though.